debian/changelog

```diff
@@ -1,3 +1,14 @@
+mercury (2.4.0~rc5-4) unstable; urgency=medium
+  [ Joseph Moore ]
+  * Update patch to na_ucx.c to add fix for connection accept.
+
+ -- Joseph Moore <joseph.moore@intel.com>  Thu, 26 Sep 2024 10:00:00 -0600
+
+mercury (2.4.0~rc5-3) unstable; urgency=medium
+  [ Brian J. Murrell ]
+  * Add --without ucx build switch
+ -- Brian J. Murrell <brian.murrell@intel.com>  Wed, Sep 04 2024 09:00:00-0600
```

Comment:
new line spacing doesn't seem consistent

Reply:
I will check this.  I added an entry to the debian/changlog for Brian's recent change (along with one for my current change), since it was left out of his PR.